blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -99,10 +217,108 @@ namespace blueprint
             return _viewManager.get();
         }
 
+        //==============================================================================
+        /**
+         * Enables "hot-reload" functionality so that ReactApplicationRoot watches JS bundle files
+         * added via evaluate() for changes. When one of these bundle files changes (i.e. from a Webpack watch/rebuild)
+         * ReactApplicationRoot will reload the bundle and update the View/ShadowView tree.
+         **/
+        void enableHotReloading(bool enable)
```

Comment:
Forgive me for maybe being nitpicky, but rather than boolean arguments in this type of situation I prefer a second method: `void enableHotReloading()` and `void disableHotReloading()`. To me it just makes the callsite so much more clear about what your intention is. Then, as I think about it, would we ever really even want a `disableHotReloading`? I can imagine in release mode not starting hot reloading, but I don't think there would be too many cases where you would start, and then want to stop, the hot reloader? What do you think?

So maybe then my ask here would be just remove the boolean argument and

```
void enableHotReloading()
{
    bundleWatcher = std::make_unique<etc>(etc);
    hotReloadingEnabled = true;
}
```

Reply:
Yep fair point. I don't really think it is necessary either.